docs/solutions/sql_server_always_on_availability_groups/terraform.tfvars

```diff
@@ -0,0 +1,18 @@
+tenancy_id = "<tenancy OCID>"
```

Comment:
All of our other examples are using "tenancy_ocid". It's useful to be consistent so that we can users can export a common set of variables for any example.

Reply:
Sounds good. I will change